sql/V001__Initial_Setup.sql

```diff
@@ -532,22 +532,22 @@ COMMENT ON TABLE semester IS '// TODO: start/end dates for site';
 --
 
 CREATE TABLE step (
-    observation_id character varying(40) NOT NULL,
-    index smallint NOT NULL,
-    instrument identifier NOT NULL,
-    step_type step_type NOT NULL
+    step_id        SERIAL                PRIMARY KEY,
```

Comment:
We were talking about using a universal ID on steps so we could associate them with datasets without having to deal with step indexing, which this gives us. So 👍 for this.